llama/sap.py

```diff
@@ -402,3 +403,27 @@ def generate_sap_control(sap_data_file: str, invoice_total: float) -> str:
     sap_control_file += "\n"
 
     return sap_control_file
+
+
+def get_and_update_sap_sequence_parameter(
```

Comment:
As discussed offline, this function should be something like `generate_next_sap_sequence_number` and should get the current parameter from SSM, extract just the sequence number, and return it incremented by 1. The function doesn't need any args, it can create the ssm class itself and the parameter key should be generated from config, e.g. `CONFIG.SSM_PATH + "SAP_SEQUENCE"` (There's already an SAP_SEQUENCE parameter set on staging ssm).